sw/vendor/riscv_compliance/riscv-target/opentitan/device/rv32imc/Makefile.include

```diff
@@ -16,11 +16,13 @@ TARGET_SIM   ?= $(OT_ROOT)/build/lowrisc_systems_top_earlgrey_verilator_0.1/sim-
 ifeq ($(OT_TARGET),fpga)
 	CARG =
 	MAKEARG =
+	NINJA_TARGET = fpga
 	PYTEST_OPT = --fpga_uart $(OT_FPGA_UART) --spiflash $(OT_ROOT)/sw/host/spiflash/spiflash \
 		--test_bin $(work_dir_isa)/$<.bin
 else
 	CARG = -DSIMULATION=1
```

Comment:
i think we actually don't need CARG, i must have forgotten to delete it.

Reply:
Done.